### PR TITLE
sc-15717 Quick view component

### PR DESCRIPTION
**Pull request by elysee15**
### Scope of changes
Fixes SC-15717

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria
image

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/projects/api/projectStatsApiService.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ApiAdapters } from '@/application/api/ApiAdapters';
+import { getValidApiResponse, Request } from '@/application/api/ApiService';
+import { APP_ROUTE } from '@/constants';
+
+import { ProjectsResponse } from '../types/projectService';
+
+export function projectStatsApiRequest(request: Request): ApiAdapters['getProjectList'] {
+  return async (tenantID: string) => {
+    console.log('[] tenantId', tenantID);
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/projects/api/projectStatsApiService.ts:9]:
```suggestion
    //console.log('[] tenantId', tenantID);
```

File: web/beacon-app/src/features/projects/loaders/projectStatsLoader.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { tenantsRequest } from '@/features/tenants/api/tenantListAPI';
+
+import { projectStatsApiRequest } from '../api/projectStatsApiService';
+
+export async function projectPageLoader() {
+  const tenants = await queryClient.fetchQuery({
+    queryFn: tenantsRequest(axiosInstance),
+  });
+
+  const projectStats = await projectStatsApiRequest(axiosInstance)(tenants.tenants[0].id);
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/projects/loaders/projectStatsLoader.ts:14]:
```suggestion
  const projectStats = await projectStatsApiRequest(axiosInstance)(tenants?.tenants[0]?.id);
```

**Review by daniellemaxwell — APPROVED** [review]
Using the react router loader is clever and I learned something new. Thanks! I noticed a spelling mistake in a file name, so I made the change and pushed it. Great work!

**Review by masskoder — APPROVED** [review]
Looks good but we are already using react-query what is the benefit to use the loader of react-router 6.4? I think the loaders are clunky to use with React Query and in our case I'm not sure it's going to add any performance value.

**Comment by elysee15** [comment]
> It looks good, but we already use react-query, so what is the benefit of using the loader of react-router 6.4? Using loaders with React Query seems clunky, and in our case, I'm not sure it will add any performance benefit over RQ.

Loader avoid using suspense to add loading ui everywhere since it is called concurrently with the component

**Pull request merged**